snuba/migrations/runner.py

```diff
@@ -494,6 +555,34 @@ def get_status(migration_key: MigrationKey) -> Status:
 
         return group_migrations
 
+    def _get_completed_migrations(
+        self, groups: Sequence[MigrationGroup]
+    ) -> List[MigrationKey]:
+        """
+        Get a list of completed migrations for a list of groups
+        """
+        migration_status = self._get_migration_status()
+
+        def get_status(migration_key: MigrationKey) -> Status:
+            return migration_status.get(migration_key, Status.NOT_STARTED)
+
+        group_migrations: List[MigrationKey] = []
+        for group in groups:
+            group_loader = get_group_loader(group)
+            for migration_id in group_loader.get_migrations():
+                migration_key = MigrationKey(group, migration_id)
+                status = get_status(migration_key)
+                if status == Status.IN_PROGRESS:
+                    # can't reverse migrations if one is stuck pending
+                    raise MigrationInProgress(str(migration_key))
+                if status == Status.NOT_STARTED:
+                    continue
+                elif status == Status.COMPLETED:
```

Comment:
the `continue` exits the loop already, so `elif` is unnecessary and can be replaced by `if`.

Reply:
updated this a bit to check for another case